### PR TITLE
Rejuvenate log levels (after implementing the setting & keywords)

### PR DESCRIPTION
**Pull request by yiming-tang-cs**
We appreciate your previous feedback and it's very helpful! Here's a reissue of californium7170 with a new version of our tool.  Again, we'd appreciate any feedback and are willing to make further changes if you wish to incorporate our PR into your project.

Settings
---
We have several analysis settings. We can vary these settings and rerun if you desire. The settings we are using in this pull request are:

- Treat ERROR/WARN levels as a category and not a traditional level, i.e., our tool ignores these log levels.
- Never lower the logging level of logging statements within catch blocks.
- Never lower the logging level of logging statements within if statements.
- Never lower the logging level of logging statements containing certain (important) keywords.
- Never raise the logging level of logging statements without particular keywords in their messages.
- Avoid log wrapping by disregarding logging statements contained in if statements mentioning log levels.

The greatest number of commits from HEAD evaluated: 1000.
The head at the time of analysis was: 785267fc8a2486260e1e65144ebaf6ca1c1ac81b

- [x] By placing an `X` in the preceding checkbox, I verify that I have signed the [Contributor License Agreement](https://github.com/SeleniumHQ/selenium/blob/master/CONTRIBUTING.md#step-6-sign-the-cla)

### CODE REVIEW

File: element-connector/src/test/java/org/eclipse/californium/elements/runner/TestRepeater.java [hunk 1]
```diff
@@ -110,7 +110,7 @@ public void run(final Runner runner, final RunNotifier notifier) {
 		if (0 == maximumRepeats) {
 			LOGGER.info("repeat until error!");
 		} else {
-			LOGGER.info("maximum repeats: {}", maximumRepeats);
+			LOGGER.debug("maximum repeats: {}", maximumRepeats);
```
Comment (yiming-tang-cs) [inline element-connector/src/test/java/org/eclipse/californium/elements/runner/TestRepeater.java:113]: We had a setting for if condition due to logging wrapping. If we consider `else` clause, it's not a logging wrapping case.

**Comment by khatchad** [comment]
I'm not sure about that. What does the UI say about the setting? Documentation?

**Comment by khatchad** [comment]
How does an if condition relate to log wrapping?

File: element-connector/src/test/java/org/eclipse/californium/elements/runner/TestRepeater.java (hunk 1 above)
Comment (yiming-tang-cs) [reply element-connector/src/test/java/org/eclipse/californium/elements/runner/TestRepeater.java:113]: Sorry, we have two settings about if-statements. The setting about log wrapping should not be the one we need here.

**Comment by yiming-tang-cs** [comment]
>I'm not sure about that. What does the UI say about the setting? Documentation?

- UI says roughly: `Never lower the logging level of logging statements inside if statements.`
- Documentation: `not lower log levels in if statements.`
- Wiki says: `Never lower the logging level of logging statements within immediate if statements without an else clause (i.e., guarded) .`
- Paper says: https://github.com/ponder-lab/Logging-Level-Evolution-Paper/blob/921108c3e21d7ae4c27ab0d17168923116bc1cec/approach.tex#L157-L176

**Pull request closed without merging**